mateo-mustapic/assignments/liferay-app/sdk/portlets/todo-portlet/docroot/js/main.js

```diff
@@ -70,7 +149,27 @@ AUI.add(
 							todoList.append(taskHtml);
 
 							taskInput.val('');
+
+							instance._countTasks();
 						}
+					},
+
+					_countTasks: function() {
```

Comment:
I would just add that method/function should do only what its name says it does. Such naming and strict rules will make code readable and easier for maintenance for other developers.
http://www.goodreads.com/book/show/3735293-clean-code